Assignments-links_using_angular(11thOct)/src/app/alert-me/alert-me.component.html

```diff
@@ -0,0 +1,2 @@
+<p>alert-me works!</p>
```

Comment:
why?

Reply:
that was intially there while component creation ,had kept it there itself for proper execution check